package.json

```diff
@@ -64,6 +64,42 @@
                 "type": "string",
                 "default": "^\\s*$",
                 "description": "The Regexp to use to match empty lines in Jumpy Line Mode"
+            },
+
+            "jumpy.fontFamily": {
+                "type": "string",
+                "default": "",
+                "description": "Font used in Jumpy decorations, defaults to font from settings"
+            },
+
+            "jumpy.fontSize": {
+                "type": "number",
+                "default": 0,
+                "description": "Font size used in Jumpy decorations, defaults to font size from settings - 1"
+            },
+
+            "jumpy.darkThemeBackground": {
+                "type": "string",
+                "default": "white",
+                "description": "Background of Jumpy decoration in dark themes"
+            },
+
+            "jumpy.darkThemeForeground": {
+                "type": "string",
+                "default": "black",
+                "description": "Text color of Jumpy decoration in dark themes"
+            },
+
+            "jumpy.lightThemeBackground": {
+                "type": "string",
+                "default": "black",
+                "description": "Background of Jumpy decoration in light themes"
+            },
+
```

Comment:
I'd remove the empty lines in `json`.

Reply:
Don't bother, repository is dead

Reply:
Looks like that. I'm started to use Acejump. The overall implementation is great. So I can recommend that extension.

https://marketplace.visualstudio.com/items?itemName=lucax88x.codeacejumper